ios/FluentUI/Tooltip/TooltipView.swift

```diff
@@ -324,6 +338,10 @@ class TooltipView: UIView {
     }
 
     private var sourcePointInAnchorView: CGPoint {
+        guard let anchorView = anchorView else {
+            preconditionFailure("Can't find anchorView")
```

Comment:
Potential concern -- do we want to crash in this case? I'd lean towards asserting (so we catch it in debug builds) and then return a default `(0,0)` value.

Reply:
Same comment for the other `preconditionFailure` cases below (`sourcePointInWindow`, `boundingRect`, `anchorViewInset(for:)`.